internal/cmd/operator-sdk/scorecard/cmd.go

```diff
@@ -91,10 +91,10 @@ If the argument holds an image tag, it must be present remotely.`,
 	scorecardCmd.Flags().DurationVarP(&c.waitTime, "wait-time", "w", 30*time.Second,
 		"seconds to wait for tests to complete. Example: 35s")
 	scorecardCmd.Flags().StringVarP(&c.storageImage, "storage-image", "b",
-		"docker.io/library/busybox@sha256:c71cb4f7e8ececaffb34037c2637dc86820e4185100e18b4d02d613a9bd772af",
+		"docker.io/bpalmer/scorecard-storage:dev",
```

Comment:
Ideally this would not be my own personal image. I *believe* I have the right permissions to build and push these images under the quay.io/operator-framework organization but wanted to wait on doing that to see what others think we should do. We could:
- push this in as is, force CI to build a new image and push it, then create a new PR to set this to the new image
- manually build and push the new images with the `latest` tag and set them here. CI *should* automatically build and push to latest for every release after that.

Reply:
I feel like the manual build/push should be okay in this case so that we get an automated CI build when this PR merges without needing a follow up PR, but I'm not strongly wedded to that option if others have an issue with it.